packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.1.json

```diff
@@ -130,9 +130,7 @@
             "type": "number"
           },
           "currency": {
-            "type": "string",
-            "minLength": 3,
-            "maxLength": 3
+            "type": "string"
```

Comment:
I'm up for keeping the minimum length constraint, to explicitely prevent "€" for instance.